### PR TITLE
feat(madara): add immortalupdates

### LINKED ISSUE #532: Immortal Updates
can you add this on source? thank you! 

https://immortalupdates.com/

### PR DESCRIPTION
**Pull request by daarkdemon**
Checklist:
- [x] Updated source's version for individual source changes
- [ ] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [ ] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device 

Closes #532

### CODE REVIEW

File: src/rust/madara/sources/immortalupdates/res/Icon.png
Comment (Skittyblock) [inline src/rust/madara/sources/immortalupdates/res/Icon.png:1]: icon should be opaque, not have a transparent background. this looks like an android icon, it should be scaled up so it's just the logo with a black background

File: src/rust/madara/sources/immortalupdates/res/filters.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+			"AND (having all of the selected genres)"
+		]
+	},
+	{
+		"type": "group",
+		"name": "Genres",
+		"filters": [
+			{
+				"type": "genre",
+				"name": " Action ",
```
Comment (Skittyblock) [inline src/rust/madara/sources/immortalupdates/res/filters.json:41]: why do all the genre names have spaces at the start and end? (probably from a generator script, but this should be fixed)

**Review by Skittyblock — CHANGES_REQUESTED** [review]

**Comment by Skittyblock** [comment]
remember to mention the issues that it closes in the description, not the title, otherwise it doesn't actually link them

File: src/rust/madara/sources/immortalupdates/res/filters.json (hunk 1 above)
Comment (daarkdemon) [reply src/rust/madara/sources/immortalupdates/res/filters.json]: fixed.

File: src/rust/madara/sources/immortalupdates/res/Icon.png
Comment (daarkdemon) [reply src/rust/madara/sources/immortalupdates/res/Icon.png:1]: fixed.

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**